tutorials/en_us/JARS/OPTIMIZATION.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+Here you will learn how to optimize you Minecraft server to get better performance.
+
+## Optimizing your .jar specific configurations to YOUR NEEDS
+
+Every .jar has its own set of configurations that you can tweak to suit your specific needs. You probably wont find as effective the config of a minigames server in a survival server. You should read every config, understand it, and choose if you need to use it or not. This is extremely important in survival servers as some config values can ruin your mob spawning rates or destroy your players custom redstone circuits.
+
+Here is a [guide](https://www.spigotmc.org/threads/guide-server-optimization⚡.283181/) made by a well-known member of the community that can help you further in this optimization, it is limited to PaperMC and doesn't have optimized configs for Tuinity, Purpur or any other forks. You can also join [this discord](https://discord.gg/yev2rN3eZH), which has an amazing Discord bot that assists you optimizing your server based on your timings report! Remember this is a general suggestion, it may not suit your needs.
```

Comment:
Should be split into multiple lines for readability

Reply:
actual lines or paragraphs?